charms/jupyter-controller/reactive/jupyter_controller.py

```diff
@@ -1,8 +1,10 @@
 from pathlib import Path
 
 import yaml
+
+from charmhelpers.core import hookenv
 from charms import layer
-from charms.reactive import hook, clear_flag, set_flag, when, when_any, when_not
+from charms.reactive import clear_flag, hook, set_flag, when, when_any, when_not
```

Comment:
I don't really have to read the rest.  Alphabetizing imports will always get a +1 out of me.